proposals/0216-widget-support.md

```diff
@@ -208,17 +202,34 @@ Today, SDL uses HMI levels to inform an app about the launch state and visibilit
 ```xml
 <function name="OnHMIStatus" functionID="OnHMIStatusID" messagetype="notification" since="1.0">
   :
-  <param name="screenID" type="Integer" mandatory="false" since="5.1" />
+  <param name="screenID" type="Integer" mandatory="false" since="5.x" />
 </function>    
 ```
 
 The additional parameter can be used by the system to specify the HMI level of a screen. If the value is omitted, it means the default screen is addressed. For other screens the screen ID specified by the app should be set in this parameter.
 
 If a widget becomes visible on the display, the HMI should notify Core that the widget is activated. Core should then notify the app that the widget is now in HMI_FULL. This can be the case if the user changes the HMI to present the widget area (e.g. the home screen shows app widgets).
 
-Audio streaming state is not related to screens but to system's audible state. To limit the changes in the RPC and support backward compatibility, `audioStreamingState` should be provided to all screens of one app. HMI and SDL core must make sure to send the same audio states to all screens. Example: If a media app has created a widget and then becomes audible, the media app should receive two `OnHMIStatus` notifications for both screens (main and widget) and both audio streaming states are set to `AUDIBLE`.
+Audio streaming state is not related to screens but to system's audible state. To limit the changes in the RPC and support backward compatibility, `audioStreamingState` should be provided to all screens of one app. HMI and SDL core should make sure to send the same audio states to all screens. Example: If a media app has created a widget and then becomes audible, the media app should receive two `OnHMIStatus` notifications for both screens (main and widget) and both audio streaming states are set to `AUDIBLE`. This will ensure a consistent audible state.
```

Comment:
I don't understand why these changes are required. Can we please focus on the revisions made?

It is supposed to be "of" because screens are entities **of** an app.